templates/oned.conf.erb

```diff
@@ -28,6 +28,9 @@
 #  scripts.
 #
 #  PORT: Port where oned will listen for xmlrpc calls.
+<% if scope.function_versioncmp([@one_version,'4.14']) >= 0 -%>
+#  LISTEN_ADDRESS: Host IP to listen on for xmlrpc calls (default: all IPs).
+<% end -%>
```

Comment:
This is comment only. I would keep the comment always in place and just add information that this is working with 4.14 and later versions only.